tests/Unit/Innovation/Utils/StringsTest.php

```diff
@@ -28,6 +28,13 @@ public function doesStringComeBeforeProvider(): array
             ['Code of Laws', 'Pottery', true], # Real example
             ['Sailing', 'São Paulo', true], # accents ignored
             ['São Paulo', 'Satellites', true], # accents ignored
+            ['HuntJ', 'Hunt‑Lenox Globe', true], # non-breaking hyphen where next letter comes after
+            ['Hunt‑Lenox Globe', 'HuntM', true], # non-breaking hyphen where next letter comes before
+            ['Abc', 'Ab\'d', true], # apostrophe where next letter comes after
+            ['Ab\'c', 'Abd', true], # apostrophe where next letter comes before
+            ['A. I.', 'Archery', true], # period and space, TODO(ARTIFACTS): Figure out right order for this.
```

Comment:
Given our discussion, we'll probably need to update the method to strip punctuation out, _then_ do the comparison